zipkin-aws/Dockerfile

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright 2015-2016 The OpenZipkin Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+#
+FROM openzipkin/zipkin:1.20.0
+MAINTAINER OpenZipkin "http://zipkin.io/"
+
+ENV ZIPKIN_AWS_REPO https://jcenter.bintray.com
+ENV ZIPKIN_AWS_VERSION 0.1.0
+
+RUN apk add unzip && \ 
```

Comment:
Need a different version of `unzip` due to a bug in the package provided on the image.  The error I encountered was `unzip: short read` when extracting the module jar.